download/index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Decred - downloads</title>
```

Comment:
Maybe I'm being overly picky here but if the title/links are going to be named "downloads" then the folder name should be "downloads" too rather than "download".